libs/design/tree/src/tree-item/tree-item.directive.ts

```diff
@@ -6,6 +6,7 @@ import {
   Inject,
   Input,
 } from '@angular/core';
+import { RouterLinkActive } from '@angular/router';
```

Comment:
This makes the `design` library dependent on `@angular/router` which I've intentionally disallowed.

Reply:
Then this feature cannot be implemented as requested. The best that we can do is set `daffTreeItem routerLinkActive="selected"` in app code.

Reply:
removed everything from design